src/app/Dashboard/components/tests/DashboardComponent.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { DashboardComponent } from '../DashboardComponent';
+
+describe('<DashboardComponent />', () => {
+  const setUpWrapper = ({
+    error = null,
+    loading = false,
+    pointsEarned = 0,
+    userActivities = [],
+    activitiesLogged = 0,
+  } = {}) => {
```

Comment:
What does line 6 to line 13 do. I can see the destructure with defaults but other than that, it is not very readable.

Reply:
Line 6 is a function that returns the shallow/mounted component. Line 13, maps parameters passed  to props. The function helps to change props passed to the shallow/mounted component. Such as in line `27, 32`